explorer_frontend/public/runtime-config.toml

```diff
@@ -4,7 +4,7 @@
 
 DOCUMENTATION_URL = "https://docs.nil.foundation/nil/intro"
 GITHUB_URL = "https://github.com/NilFoundation/nil-hardhat-example"
-API_URL = "https://explore.nil.foundation/api"
+API_URL = "http://localhost:3000/api"
```

Comment:
what is the purpose of that change? please use `runtime-config.local.toml` to override this variables. if you change it here - you need to set this vars in infra as well.